controller-mock/pkg/controller/initial_metrics_config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "metrics": [
+        {
+            "name": "motor_temperature",
+            "value": 70,
+            "range_min": 70,
+            "range_max": 85,
+            "fluct_unit": 1,
+            "strategy": "increment"
```

Comment:
Instead of strategy can we provide the time interval? This way we can showcase in the UI we are producing multiple metric data spontaneously. 

For the strategy, we can stick to increment as the initial state.

Reply:
no, the initial_metrics_config.json is the initial configuration/ starting point for the program to generate data from. The "strategy" field cannot be removed since it is critical to providing the initial direction of each individual metric movement and changing in order to track the state and direction of movement for each metric individually depending on the fluctuation unit and the range. Nonetheless, this file should not be of interest since it does not effect what is reported to Kafka and to the UI eventually.

if it is of value, there is no problem to add a timestamp to each metric on when it has been updated, but since each metric is updated every <time interval> this will not be a "spontaneous" occurrence, meaning the timestamp of each interval of metrics will essentially be the same, and will be the same as the timestamp that already exists in the controller scope